src/main/java/my/test/shop/pages/AuthPage.java

```diff
@@ -0,0 +1,62 @@
+package my.test.shop.pages;
+
+import io.qameta.allure.Step;
+import my.test.shop.driver.Driver;
+import my.test.shop.utils.Utils;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.Properties;
+
+public class AuthPage {
+
+    @FindBy(xpath = "//input[@id='email_create']")
+    private WebElement emailRegistrationFiled;
+
+    @FindBy(xpath = "//button[@name='SubmitCreate']")
+    private WebElement createAccountButton;
+
+    @FindBy(xpath = "//input[@id='email']")
+    private WebElement emailField;
+
+    @FindBy(xpath = "//input[@id='passwd']")
+    private WebElement passwordField;
+
+    @FindBy(xpath = "//button[@name='SubmitLogin']")
+    private WebElement signInButton;
+
+    private WebDriver driver;
+    private static final String LOGIN_URL = "http://automationpractice.com/index.php?controller=authentication&back=identity";
+    private static String propertyPath = "src/test/resources/email.properties";
+    private Properties properties = Utils.getProperties(propertyPath);
+
+    public AuthPage() {
+        this.driver = Driver.getInstance().getDriver();
+        PageFactory.initElements(driver, this);
+    }
+
+    @Step("Submit Email to start Registration Process")
+    public RegistrationPage submitRegistrationEmail() {
+        driver.get(LOGIN_URL);
```

Comment:
You uses this line twice in different methods. I think you can move it into constructor